operator/src/internal/cnpg/cnpg_cluster.go

```diff
@@ -31,9 +31,9 @@ func GetCnpgClusterSpec(req ctrl.Request, documentdb *dbpreview.DocumentDB, docu
 	}
 
 	// Configure storage class - use specified storage class or nil for default
-	var storageClass *string
-	if documentdb.Spec.Resource.Storage.StorageClass != "" {
```

Comment:
We are removing this line means for single cluster setup in AWS or GKE it will fail unless we are passing it right. From the controller I see we are only passing `replicationContext.StorageClass` but not from spec.